### ISSUE TITLE
Value should be a JSON object.

### ISSUE THREAD

**Issue by ashan1992** [areytsman/zabbix_ssl_template#4]
Hi,

Just got an error when ssl_ports_lld.py when discovery rule is running
![zabbix](https://user-images.githubusercontent.com/23723961/53691409-4fd78a00-3da3-11e9-9639-b2c32eb1a524.JPG)
. 

Thanks.

**Comment by areytsman** [comment]
Hi!
Run the lld script manually and check output.

**Comment by ashan1992** [comment]
Hi ,

Thanks for the reply , please check the output.
![zabbix1](https://user-images.githubusercontent.com/23723961/53691848-20794b00-3dac-11e9-8208-b414ead05113.JPG)

**Comment by areytsman** [comment]
You should use a hostname as a param.
{{ # ./ssl_ports_lld.py my-server.com }}

**Comment by ashan1992** [comment]
yes i have tried it , please check the following attachment and an after linking the template to the host there is no any trigger created to the websites.

![image](https://user-images.githubusercontent.com/23723961/53696336-e465da80-3deb-11e9-8ee2-e2df690dd8b6.png)

**Comment by areytsman** [comment]
I see an expected behavior on the screenshot. As I can see there are no ssl ports discovered on the localhost and 10.0.171.11. Do the hosts listen for SSL connections?

**Comment by ashan1992** [comment]
Yes they do.
![image](https://user-images.githubusercontent.com/23723961/53712354-82998500-3e6c-11e9-9938-16fcab76f238.png)

and sorry earlier ip was wrong and when i try with the correct ip same thing occurs.

![image](https://user-images.githubusercontent.com/23723961/53712423-c8564d80-3e6c-11e9-9c8c-a0b1af457c8b.png)



and im using Zabbix 4.0

**Comment by areytsman** [comment]
Run `openssl s_client -connect 10.171.0.7:443` and check output.
Also you can replace lines 32-35 in ssl_ports_lld.py with 
```
    except Exception as ex:
        print(ex)
        sock.close()
        return False
```
for debugging purpose and run script manually.
